samples/maven-sample/spring-petclinic/pom.xml

```
@@ -0,0 +1,248 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.springframework.samples</groupId>
+    <artifactId>spring-petclinic</artifactId>
+    <version>2.1.0.BUILD-SNAPSHOT</version>
+
+    <parent>
+        <groupId>com.epam</groupId>
+        <artifactId>maven-sample</artifactId>
+        <version>1.0</version>
+    </parent>
+    <name>petclinic</name>
+
+    <properties>
+
+        <!-- Generic properties -->
+        <java.version>1.8</java.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
+        <!-- Web dependencies -->
+        <webjars-bootstrap.version>3.3.6</webjars-bootstrap.version>
+        <webjars-jquery-ui.version>1.11.4</webjars-jquery-ui.version>
+        <webjars-jquery.version>2.2.4</webjars-jquery.version>
+        <wro4j.version>1.8.0</wro4j.version>
+        <!-- For example D:\Github\Drill-Samples\maven-sample\spring-petclinic\agent        -->
+        <pathToAgent>"Path to agent"</pathToAgent>
```

are those quotes need? Why don't use default value with TODO change? For example:
```suggestion
        <pathToAgent>"D:\Github\Drill-Samples\maven-sample\spring-petclinic\agent"</pathToAgent>
```